src/js/keyboard.js

```diff
@@ -0,0 +1,53 @@
+const constants = require('.\\constants.js');
+const processor = require('.\\processor.js');
+
+import info from '.\\infoBlock.js';
+
+class Keyboard {
+
+  constructor() {
+    if (!localStorage.getItem('kbLang')) {
+      localStorage.setItem('kbLang', 'eng');
+    }
+
+    if (!localStorage.getItem('isFirstVisit')) {
+      localStorage.setItem('isFirstVisit', 'true');
+    }
```

Comment:
the same here. we prefer to use constants instead of string literal.

Reply:
Done